src/pal/src/exception/seh-unwind.cpp

```diff
@@ -203,10 +203,12 @@ static DWORD64 GetPc(CONTEXT *context)
 {
 #if defined(_AMD64_)
     return context->Rip;
-#elif defined(_ARM64_)
+#elif defined(_ARM64_) || defined(_ARM_)
     return context->Pc;
 #else
-#error don't know how to get the program counter for this architecture
+    unreached();
```

Comment:
I would think that this should stay as #error. It is what PAL is using everywhere else, and unreached() is not really appropriate here - the code is reachable, it is just not implemented.